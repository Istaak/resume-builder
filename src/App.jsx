// Third party dependencies
import React from 'react';

// Own constants/actions/components
import './App.scss';

function App() {
  return <h1>Resume Builder</h1>;
}

export default App;
