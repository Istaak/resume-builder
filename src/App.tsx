// Third party dependencies
import { Provider } from 'react-redux';

// Own components/hooks/constants
import Router from 'router';
import { store } from 'store';
import GlobalStyles from './App.styles';

export const AppContent = () => <Router />;

const App = () => (
  <Provider store={store}>
    <GlobalStyles />
    <AppContent />
  </Provider>
);

export default App;
