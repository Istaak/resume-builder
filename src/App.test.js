// Third-party dependencies
import React from 'react';
import ReactTestRender from 'react-test-renderer';

// Own constants/actions/components
import configureStore from 'store/configure-store';
import App from './App';

describe('App: ', () => {
  test('should render correctly', () => {
    const store = configureStore();
    const component = ReactTestRender.create(<App store={store} />);
    const render = component.toJSON();
    expect(render).toMatchSnapshot();
  });
});
