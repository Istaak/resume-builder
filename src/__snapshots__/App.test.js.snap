// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App:  should render correctly 1`] = `
<h1>
  Resume Builder
</h1>
`;
